app/component/GeopositionSelector.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import connectToStores from 'fluxible-addons-react/connectToStores';
+import { FormattedMessage } from 'react-intl';
+
+import { startLocationWatch } from '../action/PositionActions';
+import PositionStore from '../store/PositionStore';
+import Icon from './Icon';
+
+const GeopositionSelector = ({ status }) => {
+  /* States:
+   * - locationing hasn't been started
+   * . locationing in progress
+   * . locationing denied
+   * . locationing failed
+   * - locationing succeeded
+   */
+  if (status === PositionStore.STATUS_NO_LOCATION) {
```

Comment:
Not all states is PositionStore are mapped, eg STATUS_GEOLOCATION_NOT_SUPPORTED. Should this be a switch statement instead to catch all cases?